### PR TITLE
feat(datasets): datasets inside projects

### PR DESCRIPTION
**Pull request by vfried**
This feature PR, it groups smaller PRs inside a feature branch

PRs included:
https://github.com/SwissDataScienceCenter/renku-ui/pull/562
https://github.com/SwissDataScienceCenter/renku-ui/pull/555

Closes:
#525
#602

### CODE REVIEW

**Comment by vfried** [comment]
The datasets are now fetched from the KG and i made a small change on the way the datasets in the overview are displayed:

image

The only information we receive at first from a dataset is the dataset name and id so we can't display the overview as before (in the card) so we will display the available datasets in a table

**Comment by vfried** [comment]
Feedback added about the status of the KG inside the datasets tab....
image

image

**Comment by rokroskar** [comment]
Will the user understand that the KG is needed to see dataset information?

**Comment by vfried** [comment]
Good point, i will add some message telling the user that the KG integration is needed to see the datasets.

**Comment by vfried** [comment]
Messages updated:

image

image

image

**Comment by vfried** [comment]
virginiatest.dev.renku.ch is up to date for testing

File: src/project/Project.present.js [hunk 1]
```diff
@@ -528,16 +529,30 @@ class ProjectViewDatasets extends Component {
...
     return [
-      <Col key="datasetsnav" sm={12} md={4} lg={2}>
-        <ProjectDatasetsNav {...this.props} />
-      </Col>
-      ,
+      kgLoading ? null
+        :<Col key="datasetsnav" sm={12} md={4} lg={2}>
+          <ProjectDatasetsNav {...this.props} />
+        </Col>,
       <Col key="datasetcontent" sm={12} md={8} lg={10}>
```
Comment (lorenzo-cavazzi) [inline src/project/Project.present.js]: The change between `md` and `lg` is big, what about making it more gradual?
`<Col key="datasetsnav" md={4} lg={3} xl={2}>`
`<Col key="datasetcontent" md={8} lg={9} xl={10}>`

File: src/project/Project.present.js [hunk 2]
```diff
@@ -603,6 +669,50 @@ function notebookLauncher(userId, accessLevel, notebookLauncher, fork, postLogin
   return (<div>{content}</div>);
 }
 
+class OverviewDatasetRow extends Component{
+  render(){
+    return  <tr>
+      <td className="align-middle">
+        <a href={this.props.fullDatasetUrl}>{this.props.name}</a>
```
Comment (lorenzo-cavazzi) [inline src/project/Project.present.js]: Should this be a `<Link>` to avoid re-loading all the page?

**Comment by vfried** [comment]
I fixed the line break issue, the file path issue seems to come from the backend, I will make an issue for this.

**Pull request merged**